ui/src/notebooks/components/Notebook.tsx

```diff
@@ -2,13 +2,26 @@ import React, {FC} from 'react'
 
 import {Page} from '@influxdata/clockface'
 import {NotebookProvider} from 'src/notebooks/context/notebook'
+import Header from 'src/notebooks/components/Header'
```

Comment:
Nit here and I know it _feels_ redundant.  But searching for `NotebookHeader` vs `Header` is much easier

Reply:
lol i just switched all of these components from `NotebookComponent` to `Component` as there was a `src/notebooks/components/` + `NOTEBOOK EVERYTHING` folder that made it hard to find things locally. i can switch it back

Reply:
k. this is gonna show up in a following PR after i get all this first wave out. just mapped out the sequencing